test/e2e/serial/tests/workload_placement_tmpol.go

```diff
@@ -1778,7 +1746,7 @@ func isRequestingHugepages(podRes podResourcesRequest) bool {
 func newPodScopeTMPolicyFuncs() tmPolicyFuncs {
 	return tmPolicyFuncs{
 		name:                    func() nrtv1alpha1.TopologyManagerPolicy { return nrtv1alpha1.SingleNUMANodePodLevel },
-		setupPadding:            setupPaddingPodLevel,
+		setupPadding:            setupPaddingLevel,
```

Comment:
we should remove this field and call the now-universal `setupPadding` function directly in the call sites